src/page/order-summary.page.ts

```diff
@@ -0,0 +1,12 @@
+import { $ } from 'protractor';
+
+export class OrderSummaryPage {
+
+  constructor () {
+  }
+
+  public async goToOrder(): Promise<void> {
+    await expect($('#center_column > div > p > strong').getText())
```

Comment:
The idea with POM is just encapsulating business behaviors; verifications or expects should be done just in test classes. Please create an attribute for the confirmation text ant make this function return that value